internal/core/project.go

```diff
@@ -392,10 +392,7 @@ func (p *Project) DefaultProvider(opts *core.DefaultProviderOptions) (string, er
 		if opts.CheckUsable {
 			logger.Debug("Checking usable on provider", "provider", pp.Name)
 			pluginImpl := plug.Plugin.(core.Provider)
-			usable, err := pluginImpl.Usable()
-			if err != nil {
-				return "", err
-			}
```

Comment:
I don't think we want to ignore this error here given that it will indicate there is something wrong with the plugin and we'll want to bail and report the error if that is the case.